CUSTOMIZE.md

```diff
@@ -94,7 +94,7 @@ Here's a minimal example:
 
 ```python
 # keybinding/keybinding.py
-class Keybinding(object):
+class Keybinding():
```

Comment:
https://docs.python.org/3/tutorial/classes.html#class-definition-syntax seems to suggest we go even further and do `class Keybinding:`

Perhaps we should adjust the rest of these to match that syntax?

Reply:
Ah, good point. Yes, I'll make that change.